uwblueprint/jump-math
# Add empty state for individual classrooms page tabs

## Notion ticket link

[Add empty state for archived classrooms](https://www.notion.so/uwblueprintexecs/Add-empty-state-for-archived-classrooms-793f75308ae748daafb56d5471a967ad)



## Implementation description
Adds empty states for individual tabs on the classrooms page, in case one tab is empty and the other is not. I added a new empty section for archived classrooms and reused the existing empty classrooms state for active classrooms.


User with no archived classes but at least one active:
<img width="1438" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/3e243e6d-1c4b-4549-be0e-c80184f19b96">



User with no active classes but at least one archived:
<img width="1444" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/68762764-3490-4443-9aad-748089b05629">


User with no classes at all (existing page behaviour):
<img width="1436" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/85320ec2-b6e4-474f-aaa6-cb5c927821d4">




## Steps to test
1. Make sure you have at least one archived and one active classroom.
2. Delete all your active classrooms and view the empty state for that tab.
3. Delete all your archived classrooms and view the overall empty state.
4. Add an active classroom and view the archived tab empty state.



## What should reviewers focus on?
* 


## Checklist
